scripts/cleanup_cloud.rb

```diff
@@ -0,0 +1,36 @@
+require_relative '../lib/foreman_google/google_compute_adapter'
+
+path = ARGV[0]
+zone = ARGV[1]
+
+if path.nil? || zone.nil?
+  puts 'Missing argument path or zone'
+  puts "Run script like this: bundle exec ruby scripts/cleanup_cloud.rb 'path/to/json' 'zone_name'"
+  exit 1
+end
+
+client = ForemanGoogle::GoogleComputeAdapter.new auth_json_string: File.read(path)
+
+instances = client.instances(zone)
+
+puts 'Found instances:'
+puts 'Nothing found' if instances.empty?
+
+instances.each do |instance|
+  puts ">>> Deleting #{instance.name} instance"
+  client.delete_instance(zone, instance.name)
```

Comment:
It feels like deletion of an instance is an asynchronous operation and disk removal may fail if the instance removal is not fully finished yet.

```
$ bundle exec ruby scripts/cleanup_cloud.rb  /home/aruzicka/Downloads/noted-terra-303613-69ae49292458.json     us-central1-a
Found instances:
>>> Deleting instance-1 instance

Found disks:
>>> Deleting disk-1 disk
Traceback (most recent call last):
        17: from scripts/cleanup_cloud.rb:30:in `<main>'
        16: from scripts/cleanup_cloud.rb:30:in `each'
        15: from scripts/cleanup_cloud.rb:32:in `block in <main>'
        14: from /home/aruzicka/vcs/foreman/foreman-google/app/lib/foreman_google/google_compute_adapter.rb:76:in `delete_disk'
        13: from /home/aruzicka/vcs/foreman/foreman-google/app/lib/foreman_google/google_compute_adapter.rb:115:in `delete'
        12: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/google-cloud-compute-v1-1.3.0/lib/google/cloud/compute/v1/disks/rest/client.rb:469:in `delete'
        11: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/google-cloud-compute-v1-1.3.0/lib/google/cloud/compute/v1/disks/rest/service_stub.rb:202:in `delete'
        10: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/gapic-common-0.9.0/lib/gapic/rest/client_stub.rb:76:in `make_delete_request'
         9: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/gapic-common-0.9.0/lib/gapic/rest/client_stub.rb:129:in `make_http_request'
         8: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/faraday-1.10.0/lib/faraday/connection.rb:200:in `delete'
         7: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/faraday-1.10.0/lib/faraday/connection.rb:516:in `run_request'
         6: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/faraday-1.10.0/lib/faraday/rack_builder.rb:154:in `build_response'
         5: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/gapic-common-0.9.0/lib/gapic/rest/faraday_middleware.rb:38:in `call'
         4: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/faraday-retry-1.0.3/lib/faraday/retry/middleware.rb:140:in `call'
         3: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/faraday-1.10.0/lib/faraday/middleware.rb:18:in `call'
         2: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/faraday-1.10.0/lib/faraday/response.rb:61:in `on_complete'
         1: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/faraday-1.10.0/lib/faraday/middleware.rb:19:in `block in call'
/home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/faraday-1.10.0/lib/faraday/response/raise_error.rb:16:in `on_complete': the server responded with status 400 (Faraday::BadRequestError)
        6: from scripts/cleanup_cloud.rb:30:in `<main>'
        5: from scripts/cleanup_cloud.rb:30:in `each'
        4: from scripts/cleanup_cloud.rb:32:in `block in <main>'
        3: from /home/aruzicka/vcs/foreman/foreman-google/app/lib/foreman_google/google_compute_adapter.rb:76:in `delete_disk'
        2: from /home/aruzicka/vcs/foreman/foreman-google/app/lib/foreman_google/google_compute_adapter.rb:115:in `delete'
        1: from /home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/google-cloud-compute-v1-1.3.0/lib/google/cloud/compute/v1/disks/rest/client.rb:446:in `delete'
/home/aruzicka/.toolbox/f35/.rvm/gems/ruby-2.7.2/gems/google-cloud-compute-v1-1.3.0/lib/google/cloud/compute/v1/disks/rest/client.rb:484:in `rescue in delete': An error has occurred when making a REST request: The disk resource 'projects/noted-terra-303613/zones/us-central1-a/disks/disk-1' is already being used by 'projects/noted-terra-303613/zones/us-central1-a/instances/instance-1' (Google::Cloud::InvalidArgumentError)
        5: from scripts/cleanup_cloud.rb:30:in `<main>'
        4: from scripts/cleanup_cloud.rb:30:in `each'
        3: from scripts/cleanup_cloud.rb:32:in `block in <main>'
        2: from /home/aruzicka/vcs/foreman/foreman-google/app/lib/foreman_google/google_compute_adapter.rb:76:in `delete_disk'
        1: from /home/aruzicka/vcs/foreman/foreman-google/app/lib/foreman_google/google_compute_adapter.rb:114:in `delete'
/home/aruzicka/vcs/foreman/foreman-google/app/lib/foreman_google/google_compute_adapter.rb:117:in `rescue in delete': uninitialized constant ForemanGoogle::GoogleComputeAdapter::Foreman (NameError)
```

+ the name error is something we should address